app/api/activitylog/activitylogMiddleware.js

```diff
@@ -38,13 +38,14 @@ export default (req, _res, next) => {
   if (mustBeLogged(baseurl, method, body)) {
     const time = Date.now();
     const expireAt = date.addYearsToCurrentDate(1);
-    if (body.password) body.password = '*****';
+    const bodyLog = { ...body };
+    if (bodyLog.password) bodyLog.password = '*****';
```

Comment:
@grafitto there is a max-statements issue on codeclimate for this method, since all new lines are required, maybe we could reduce the old code by instance removing the constant for time and using it inline at the entry `time: Date.now()`, what do you think?

Reply:
Nice idea, will do that

Reply:
I believe the tests are already testing the password as opposed to `random()`

Reply:
Done